Sources/Common/CustomerIOInstance.swift

```diff
@@ -144,185 +80,7 @@ public class CustomerIO: CustomerIOInstance {
-        implementation?.registeredDeviceToken
-    }
-
     public var config: SdkConfig? {
         implementation?.config
     }
-
-    /**
-     Modify attributes to an already identified profile.
-
-     Note: The getter of this field returns an empty dictionary. This is a setter only field.
-     */
-    public var profileAttributes: [String: Any] {
-        get {
-            implementation?.profileAttributes ?? [:]
-        }
-        set {
-            implementation?.profileAttributes = newValue
-        }
-    }
-
-    /**
-     Use `deviceAttributes` to provide additional and custom device attributes
-     apart from the ones the SDK is programmed to send to customer workspace.
-
-     Example use:
-     ```
-     CustomerIO.shared.deviceAttributes = ["foo" : "bar"]
-     ```
-     */
-    public var deviceAttributes: [String: Any] {
-        get {
-            implementation?.deviceAttributes ?? [:]
-        }
-        set {
-            implementation?.deviceAttributes = newValue
-        }
-    }
-
-    /**
-     Identify a customer (aka: Add or update a profile).
-
-     [Learn more](https://customer.io/docs/identifying-people/) about identifying a customer in Customer.io
-
-     Note: You can only identify 1 profile at a time in your SDK. If you call this function multiple times,
-     the previously identified profile will be removed. Only the latest identified customer is persisted.
-
-     - Parameters:
-     - identifier: ID you want to assign to the customer.
-     This value can be an internal ID that your system uses or an email address.
-     [Learn more](https://customer.io/docs/api/#operation/identify)
-     - body: Request body of identifying profile. Use to define user attributes.
-     */
-    public func identify<RequestBody: Codable>(
-        identifier: String,
-        body: RequestBody
-    ) {
-        // This code handles a specific scenario where a user
-        // unexpectedly provides a `nil` value for the RequestBody
-        // parameter, which the method isn't designed to handle directly.
-        // To prevent errors and ensure compatibility, this condition checks
-        // if the `body` is `nil`. If it is, it substitutes it with an
-        // EmptyRequestBody() object before passing it along to the
-        // identify(). This replacement safeguards against possible
-        // issues that could arise(eg. app crash). While the `identify`
-        // method itself provide polymorphic capabilities to handle `nil`,
-        // this specific check within this method offers an additional
-        // layer of protection and clarity for this case.
-        // (refer https://github.com/customerio/customerio-ios/blob/94cbf686c3c2a405534cfe908f7166558a3e0b5d/Sources/Tracking/CustomerIO.swift#L71-L75)
-        if let optionalValue = body as? Any?, optionalValue == nil {
-            implementation?.identify(identifier: identifier, body: EmptyRequestBody())
-            return
-        }
```

Comment:
Do we still need this workaround? I remember it caused sample app to crash on Random Login, but it so far is working without it. Any other way to test and confirm this?

Reply:
It is crashing on using Datapipeline methods too. This will be addressed in **MBL-109** !

Reply:
Yes, but we may not need tracking's identify method now, so this solution is not needed here in this case.

Reply:
I too think the same, this might not be needed now. However, if still required, we can update it in `MBL-109`